docs/book/src/managed/managed.md

```diff
@@ -0,0 +1,40 @@
+# Managed Clusters
+
+This section contains information specific to configuring managed (AKS) Kubernetes clusters using Cluster API Provider for Azure (CAPZ).
+See [self-managed clusters](../self-managed/self-managed.md) for information specific to provisioning these clusters.
+
+Documents under the Managed Clusters area:
+- [Adopting Clusters](managed.md)
+- [ASO Managed Clusters](asomanagedcluster.md)
+- [Managed Clusters](managedcluster.md)
+- [Managed Clusters - Join VMSS Nodes](adopting-clusters.md)
+- [Troubleshooting](troubleshooting.md)
+
+## CAPZ's Managed Cluster versus CAPZ's ASOManaged Cluster
+
+There are two APIs which can create AKS clusters and it's important to understand the differences.
+
+**ManagedCluster** - is the original API to provision AKS clusters introduced with the [0.4.4 release](https://github.com/kubernetes-sigs/cluster-api-provider-azure/releases/tag/v0.4.4).  This more closely matches the CAPI style for YAML code for other providers. The original code was based on directly using the Azure Go SDK, but in the [1.11.0 release](https://github.com/kubernetes-sigs/cluster-api-provider-azure/releases/tag/v1.11.0) was switched to utilize Azure Service Operator (ASO) as a dependency for provisioning.  This supports the preview API, but does not natively support all AKS features available.  The ManagedCluster API will eventually be deprecated, [see the roadmap for more information](../roadmap.md).
+
+**ASOManagedCluster** - was created in the [1.15 release of CAPZ](https://github.com/kubernetes-sigs/cluster-api-provider-azure/releases/tag/v1.15.0) and creates a CAPI-compliant wrapper around the existing ASO definitions for AKS clusters.  It has 100% API coverage for the preview and current AKS APIs via the ASO AKS CRDs.  This is the long-term plan to support provisioning AKS clusters using CAPZ.  [See the roadmap for more information](../roadmap.md)
+
+### Why is ASOManagedCluster the future?
+
+The biggest challenge with ManagedCluster is attempting to keep up with the velocity of feature changes to the frequently changing AKS API.  This model requires every single feature to be added into code directly.  Even with the simplification of code to utilize ASO as a dependency, there still is quite a bit of time required to keep up with these features.  Ulitimately, this is an unsustainable path long-term.  The [asoManaged*Patches feature](managedcluster.md#warning-warning-this-is-meant-to-be-used-sparingly-to-enable-features-for-development-and-testing-that-are-not-otherwise-represented-in-the-capz-api-misconfiguration-that-conflicts-with-capzs-normal-mode-of-operation-is-possible) enables patching/adding various AKS fields via ASO to help try to fill some of these gaps, but this comes at the cost of unknown full testing.
```

Comment:
Is there a reason why asoManagedPatches here has an asterisk? Just making sure it's a typo and not something else.

Reply:
There are a few names inbetween asoManaged and Patches which is why I put the *.  I could spell out each option, but then it would be longer.